extension/src/plots/paths/model.ts

```diff
@@ -144,6 +151,30 @@ export class PathsModel extends PathSelectionModel<PlotPath> {
     return this.data.length > 0
   }
 
+  public setHasCustomSelection(hasCustomSelection: boolean) {
+    this.hasCustomSelection = hasCustomSelection
+    this.persist(
+      PersistenceKey.PLOTS_HAS_CUSTOM_SELECTION,
+      this.hasCustomSelection
+    )
+  }
+
+  public checkIfHasPreviousCustomSelection() {
+    if (this.hasCustomSelection === undefined) {
+      const statuses = this.getTerminalNodeStatuses()
+      const plotsLength = statuses.length
+
+      const hasCustomSelection =
+        plotsLength > 20
+          ? plotsLength -
+              statuses.filter(nodeStatus => nodeStatus !== Status.SELECTED)
```

Comment:
Minor! Should we move all or part of `plotsLength -
              statuses.filter(nodeStatus => nodeStatus !== Status.SELECTED).length !==
            20` into a variable? Personally, I had to reread the statement multiple times to figure how what it was doing :)